### PR TITLE
Adding a test to validate loading images

### PR DESCRIPTION
**Pull request by mattleibow**
### Description of Change

I noticed that in one of my PRs I loaded a lower res image and another test caught it. Adding these tests to ensure that at least we validate the on-screen image.

Depends on:

 - https://github.com/dotnet/maui/pull/20169

### CODE REVIEW

File: src/Controls/tests/UITests/Tests/Concepts/ImageLoadingGalleryTests.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		{
+		}
+
+		protected override void NavigateToGallery()
+		{
+			App.NavigateToGallery("Image Loading Gallery");
+		}
+
+		[Test]
+		public void LoadAndVerifyPng() => LoadAndVerify(Test.ImageLoading.FromBundlePng);
```
Comment (jsuarezruiz) [inline src/Controls/tests/UITests/Tests/Concepts/ImageLoadingGalleryTests.cs:21]: Nice!. Need to include the Windows snapshots.
image

Comment (mattleibow) [reply src/Controls/tests/UITests/Tests/Concepts/ImageLoadingGalleryTests.cs:21]: that was windows just blowing up for some reason, rerunning in hopes and dreams

**Comment by mattleibow** [comment]
Waiting on https://github.com/dotnet/maui/pull/20169/files

**Pull request merged**